extensions/nns/build.rs

```diff
@@ -0,0 +1,36 @@
+use std::path::PathBuf;
+
+const REPLICA_REV: &str = "90e2799c255733409d0e61682685afcc2431c928";
+
+const BINARY_DEPENDENCIES: &[(&str, &str)] = &[
+    // (downloaded binary name, renamed binary name)
+    ("ic-nns-init", "ic-nns-init"),
+    ("ic-admin", "ic-admin"),
+    ("sns", "sns-cli"),
+];
+
+fn main() {
+    // keep copy of the dependency in the root of the project, so that cargo-dist will be able to package it into a tarball
+    let manifest_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+    // and also in `target/debug` or `target/release` for development purposes (e.g. cargo run)
+    let target_dir = manifest_dir
+        .parent()
+        .unwrap()
+        .parent()
+        .unwrap()
+        .join("target")
+        .join(std::env::var("PROFILE").unwrap());
+    for (binary_name, renamed_binary_name) in BINARY_DEPENDENCIES {
+        let destination_paths = (
+            manifest_dir.join(renamed_binary_name),
+            target_dir.join(renamed_binary_name),
+        );
```

Comment:
We can improve readability by giving meaningful names to `destination_paths.0` and `destination_paths.1`.  This construct as-written is essentially `let local_variables = (first local variable, second local variable)` and then accessing `local_variables.0` and `local_variables.1`.

Also, not to put too fine a point on it, but the final act of this method is to copy one "destination path" to the other "destination path".

I leave the details to the PR author.

```suggestion
        let some_meaningful_name = manifest_dir.join(renamed_binary_name);
        let some_other_meaningful_name = target_dir.join(renamed_binary_name);
```